app/ui/src/app/customizations/tech-extensions/detail/tech-extension-detail.component.ts

```diff
@@ -1,7 +1,8 @@
 import { Component, OnInit, ViewChild } from '@angular/core';
 import { ActivatedRoute, Router } from '@angular/router';
 import { Observable } from 'rxjs/Observable';
-import { Extension, Integrations } from '../../../model';
+import { Extension } from '../../../model';
```

Comment:
Since `model.ts` is located at the very top of the `app` folder, it an be safely reached through the `@syndesis/ui/model` with no fear to circular dependencies.